zerver/views/streams.py

```diff
@@ -269,7 +269,9 @@ def update_stream_backend(
     message_retention_days: Optional[Union[int, str]] = REQ(
         json_validator=check_string_or_int, default=None
     ),
-    can_remove_subscribers_group_id: Optional[int] = REQ(json_validator=check_int, default=None),
+    can_remove_subscribers_group: Optional[int] = REQ(
+        "can_remove_subscribers_group_id", json_validator=check_int, default=None
```

Comment:
I think for type reasons I'd prefer to keep the variable called `_id` if it's an integer, and compute a `setting_group_id_name` variable in the loop below.